04_numeronym/n0npax/main.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func shorten(word string) string {
+	if len(word) < 4 {
```

Comment:
Could potentially use a local variable for length I notice it is derived several times.

Reply:
Thanks for this. Fixed